pydarn/plotting/power.py

```diff
@@ -189,7 +189,7 @@ def plot_pwr0_statistic(cls, records: List[dict], beam_num: int = 0,
 
```

Comment:
same as in acf, is this new for this PR or included in the PR for this new plotting function?

Reply:
No comments on it from me if it's just a tweak for this pr.

Reply:
Included small bug fix like ACF